src/main/java/org/cms/course/Course.java

```diff
@@ -0,0 +1,44 @@
+package org.cms.course;
+
+public class Course {
+
+    private int id;
+    private String name;
+    private String description;
+
+    public Course(){
+
+    }
+
+    public Course(int id, String name, String description) {
+        super();
```

Comment:
Do we need it?

Reply:
Not as of now. It came along with the boiler plate code. :-)